osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -98,7 +98,7 @@ private double computeAimValue()
 
             // Combo scaling
             if (Attributes.MaxCombo > 0)
-                aimValue *= Math.Min(Math.Pow(scoreMaxCombo, 0.8) / Math.Pow(Attributes.MaxCombo, 0.8), 1.0);
+                aimValue *= (Math.Exp(-Math.Exp((930.58 - scoreMaxCombo) / 969.16)) - 0.073373) / (Math.Exp(-Math.Exp((930.58 - Attributes.MaxCombo) / 969.16)) - 0.073373);
```

Comment:
From a raw code standpoint I'd probably split this formula to a method and extract out the constants so that they're not repeated all over.

You could probably even make them `private static readonly` fields and use the formulae for the parameters in code directly in the initialiser, in case there's a need for potential future tweaking.

Reply:
I've tried to change it. Is it now how you mean? Forgive me I'm quite inexperienced with this stuff.

Reply:
Seems better, thanks.